src/web/components/elements/TextBlockComponent.tsx

```diff
@@ -149,13 +150,24 @@ const paraStyles = css`
 		position: relative;
 		vertical-align: baseline;
 	}
+
+	[data-dcr-style='bullet'] {
+		display: inline-block;
+		content: '';
+		border-radius: 0.375rem;
+		height: 0.75rem;
+		width: 0.75rem;
+		margin-right: 0.125rem;
+		background-color: ${palette.background.avatar};
```

Comment:
`palette.background.avatar` is the wrong property to use. The risk with picking up values for other areas of the page is when they change (and they will) that decision will bleed into unrelated areas of the code. It's better to add in a specific decision function for this use case. Like `palette.fill.bullet` or maybe `palette.background.bullet`?